doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,563 @@
+
+2. Functional approach with loaning
+
+    ```cpp
+    publisher.loan(sizeof(CounterTopic))
+        .and_then([&](iox::popo::Sample<void>& sample) {
+            new (sample.get()) CounterTopic(73);
+            sample.publish();
+        })
+        .or_else([](iox::popo::AllocationError) {
+            /* handle the error */
+        });   
+    ```
+    Notice that we get an untyped sample, ``iox::popo::Sample<const void>``. We could also use ``auto& sample`` in the
+    lambda arguments to shorten it. Again we access the pointer to the underlying raw memory of the sample and
+    construct the data we want to send.
+
+
+
+#### Creating a subscriber
+
+While the string identifiers still have to match those in the publisher, as in the untyped publisher there is no
+template type argument.
+
+```cpp
+iox::popo::UntypedSubscriber subscriber({"Group", "Instance", "CounterTopic"});
+subscriber.subscribe();
+```
+#### Receiving data
+
+Receiving the data works in the same way as in the typed API, the main difference is the ``reinterpret_cast`` needed
+before accessing the data (since the subscriber has no knowledge of the underlying type).
+
+
+```cpp
+while (keepRunning)
+{
+    // wait for new data (either sleep and wake up periodically or by notification from the waitset)
+
+    subscriber->take()
+        .and_then([](iox::popo::Sample<const void>& sample) {
+            CounterTopic* ptr = reinterpret_cast<CounterTopic*>(sample.get());
+            /* process the received data using the ptr */
+        })
+        .if_empty([] { /* no data received but also no error */ })
+        .or_else([](iox::popo::ChunkReceiveError) { /* handle the error */ });
+}
+```
+
+Note that since the received sample received is untyped (``iox::popo::Sample<const void>``), we cannot use
+``operator->`` to access the members of the underlying type but have to cast it to the correct type ``CounterTopic``
+manually.
+
+As in the untyped case we also could use a loop to get all samples as long as they are available.
+
+### Shutdown
+
+Once we are done sending, we call ``stopOffer`` at the publisher.
+
+```cpp
+publisher.stopOffer();
+```
+
+Similarly the subscriber can ``unsubscribe`` to stop receiving any data.
+
+```cpp
+subscriber.unsubscribe();
+```
+
+Both will also be called in the respective destructor if needed (i.e. if the publisher is still offering or the
+subscriber is still subscribed).
+
+### Running an iceoryx system
+
+Now that we have applications capable of sending and receiving data, we can run the complete iceoryx system.
+
+First we need to start Roudi.
+
+    # If installed and available in PATH environment variable
+    iox-roudi
+    # If build from scratch with script in tools
+    $ICEORYX_ROOT/build/posh/iox-roudi
+
+Afterwards we can start the applications which immediately connect to the Roudi via their runtime.
+
+When the applications terminates the runtime cleans up all resources needed for communication with Roudi. This
```

Comment:
either ``application`` or ``terminate``

Reply:
I think this was a misunderstanding. I think the sentence is correct Added a comma and an "s", though `When the application terminates, the runtime cleans up all resources`